requirements.txt

```diff
@@ -7,6 +7,7 @@ Keras==2.1.5
 matplotlib
 nibabel
 numpy
+onnxruntime==1.4.0
```

Comment:
Not a huge fan of adding this, but at the same time I don't know what else is better than having the issue fixed upstream.

Anyways, is it worth adding the platform check too? `sys_platform == "darwin"`?

Reply:
Maybe something like what is done for torch ? (line 28)

Reply:
> Not a huge fan of adding this, but at the same time I don't know what else is better

I agree. I'll add a comment making it clear that this should be a temporary measure, rather than a long-term fix, and to refer back to the issue to check if it's safe to revert.

> having the issue fixed upstream.

By upstream here, do you mean pinning it in IVADOMED instead? Or do you mean even further upstream (i.e. opening an issue with `onnxruntime` and troubleshooting with them)? 

I'm curious about the opening an issue, because the error we're getting has come up on `onnxruntime` issues before, just in different contexts. So perhaps they might have some insight as to why this is happening in the first place.

> Maybe something like what is done for torch ? (line 28)

I think that's a good idea. :)